docs/station-rewards.md

```diff
@@ -3,8 +3,8 @@
 Stations are rewarded with $WXM tokens (governance participation) based on the data they contribute on a daily basis.
 
 ::: info
-- Stations will start receiveing $WXM rewards on Arbitrum **mainnet** in February 2024.
 - Stations will start receiving $WXM rewards on Arbitrum **sepolia testnet** by January 16, 2024
```

Comment:
`...have started receiving WXM rewards on Arbitrum **sepolia testnet** since January 2024`

Reply:
I'm approving this for now to proceed with the announcement (as it is blocked by this list) and this is a minor change that can be done later :)